chcemvediet/apps/anonymization/cron.py

```diff
@@ -25,6 +33,47 @@ def attachment_normalization():
             content_type=content_types.PDF_CONTENT_TYPE
         )
         cron_logger.info(u'Normalized attachment: {}'.format(attachment))
+
+    elif attachment.content_type in content_types.LIBREOFFICE_CONTENT_TYPES:
+        directory_name = tempfile.mkdtemp(dir=u'.')
+        file_copy = os.path.join(directory_name, u'file' + guess_extension(attachment.content_type))
+        shutil.copy2(attachment.file.path, file_copy)
+
+        p = subprocess32.Popen(
```

Comment:
Je nutne pouzivat `subprocess32.Popen` a `p.communicate` potom handlovat rucne? Neni jednoduchsie pouzit `subprocess.run()`? Ci to nieco nevie, co potrebujeme?

Reply:
:ok: